Mathlib/Algebra/Group/Commute/Defs.lean

```diff
@@ -44,6 +44,9 @@ section Mul
 
 variable {S : Type*} [Mul S]
 
+@[to_additive]
+theorem _root_.commute_def {a b : S} : Commute a b ↔ a * b = b * a := .rfl
```

Comment:
That's just the rewriting lemma for `Commute`, right?

Reply:
No, you also need to unfold `SemiconjBy`

Reply:
You can use a `Commute a b` inside of a `rw`, but you can't easily `rw` in a `Commute a b` without first unfolding `Commute` *and* `SemiConjBy`